jarbas/core/querysets.py

```diff
@@ -29,6 +29,9 @@ def list_distinct(self, field, order_by_field, query=None):
         self = self.values(field, order_by_field).order_by(order_by_field)
         return self.distinct()
 
+    def receipt(self):
```

Comment:
What about `has_receipt_url` to be clearer? There is so much info about receipts one can get confused (fetched? url? text?)…